docs/idle-crafting.rst

```diff
@@ -0,0 +1,56 @@
+idle-crafting
+=============
+
+.. dfhack-tool::
+    :summary: Allow dwarfs to automatically satisfy their need to craft objects.
+    :tags: fort gameplay
+
+This script allows dwarves to automatically satisfy their crafting needs. The
+script is configured through an overlay that is added to the main page of
+craftsdwarf's workshops, as described below.
+
+Usage
+-----
+
+::
+
+    idle-crafting status
+    idle-crafting disable
+    disable idle-crafting
+
+The ``status`` command prints statistics about the status of the tool and the
+satisfaction of "craft item" needs in your fort. Both variants of the
+``disable`` command disallow idle crafting at all workshops and disable the
+tool.
+
+Examples
+--------
+
+``idle-crafting -t 10000,1000,500 status``
+    Reset all thresholds to defaults and print status information.
+
+Options
+-------
+
+``-t <number list>``, ``--thresholds <number list>``
```

Comment:
I think this would be better represented as a subcommand instead of an option, since it takes a specific action that is logically distinct (I feel) from "status"